servers/visual_server.cpp

```diff
@@ -283,7 +283,7 @@ RID VisualServer::make_sphere_mesh(int p_lats, int p_lons, float p_radius) {
 	return mesh;
 }
 
-RID VisualServer::material_2d_get(bool p_shaded, bool p_transparent, bool p_cut_alpha, bool p_opaque_prepass) {
+RID VisualServer::material_2d_get(bool p_shaded, bool p_transparent, bool p_cut_alpha, bool p_opaque_prepass, bool p_double_sided) {
```

Comment:
I added this parameter to the end, because it's new. But perhaps it should come after `p_transparent`, to group it with the other draw flags?